package.json

```diff
@@ -15,7 +15,7 @@
     "@ckeditor/ckeditor5-dev-webpack-plugin": "^2.0.22",
     "@ckeditor/ckeditor5-editor-classic": "^1.0.0-alpha.2",
     "@ckeditor/ckeditor5-essentials": "^1.0.0-alpha.2",
-    "@ckeditor/ckeditor5-heading": "vladikoff/ckeditor5-heading#short-heading",
+    "@ckeditor/ckeditor5-heading": "Natim/ckeditor5-heading#short-heading",
```

Comment:
Refs https://github.com/vladikoff/ckeditor5-heading/pull/1

Reply:
Merged PR!

Reply:
Should we move your repository to mozilla organization?